### ISSUE TITLE
sim title is improperly cased

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/charges-and-fields#165]
This is a minor nit about the title of this sim, "Charges And Fields".   

For titlecase in general, the word "and" should be lowercase, i.e. "Charges and Fields".   

Other sims appear to follow the titlecase convention; "Balloons and Static Electricity", "Energy Forms and Changes", "Isotopes and Atomic Mass", "Masses and Springs", ...

**Comment by arouinfar** [comment]
YES! This has bothered me for awhile now, and was reported in #152 (I believe we also noticed this not long after 1.0 but for priority reasons, decided to punt on it). This discrepancy in casing can cause some strange issues on the website. For example the table of sims under Desktop/Laptop Computer on the [Offline Access](https://phet.colorado.edu/en/offline-access) page lists the sim twice.

![image](https://user-images.githubusercontent.com/8419308/61316296-eee2a880-a7bd-11e9-9e5f-e561da889d81.png)

The HTML5 version of Gravity and Orbits is also improperly cased, so if we decide to fix this sim, we should do the same for GAO.

**Comment by ariel-phet** [comment]
@pixelzoom is this something that can be fixed on the website, or does it need to be fixed in sim code as a "simple" maintenance release?

**Comment by pixelzoom** [comment]
It needs to be changed in `charges-and-fields-strings_en.json` in master:

```js
  "charges-and-fields.title": {
    "value": "Charges And Fields"
  },
```

If you want a maintenance release, also `change charges-and-fields-strings_en.json` in the current release branch, then republish.  

I don't know where the title that appears on the website comes from.

**Comment by ariel-phet** [comment]
@pixelzoom do you mind doing this Maintenance release? Seems fairly straightforward. I am pretty confident that the sim title on the website comes from the sim strings (so that way there is not a conflict between title of the sims and title on the website or translated versions for such).

**Comment by ariel-phet** [comment]
And at low priority, when ever you feel like ticking something off, no rush.

**Comment by pixelzoom** [comment]
Done:
* Patched in master and 1.0 branch.
* Tested in 1.0.42-rc.1.
* Published in 1.0.42.
* Tested the published version and a few translations. 
* Confirmed that the title changed on the website.

@ariel-phet FYI.  Closing.
